bandwidth/webrtc/utils/transfer_util.py

```diff
@@ -6,14 +6,20 @@
 @copyright Bandwidth INC
 """
 
-def generate_transfer_bxml(device_token, sip_uri='sip:sipx.webrtc.bandwidth.com:5060'):
+def generate_transfer_bxml(device_token, voice_call_id, sip_uri='sip:sipx.webrtc.bandwidth.com:5060'):
     """
     Returns BXML string with WebRTC a device token to perform a SIP transfer
+    :param device_token: The device token
+    :param voice_call_id: The voice call id, used for debugging
```

Comment:
Maybe call this the Bandwidth Voice API Call ID being transferred... I wouldn't worry about specifying it being for debugging (customers don't need to know that :) )

Reply:
Good point - I'll clarify thanks!